admin/templates/installation/qserv-meta.conf

```diff
@@ -59,6 +59,12 @@ master = 127.0.0.1
 # master = qserv-master-example-name.mydomain.edu
 
 
+[wmgr]
+
+# Port number for worker management service
+port = 5012
```

Comment:
Good integration in the configuration framework. It shows you've understood it well. So I'm interested if you have ideas to improve it. But not in this ticket of course :-)